tests/ilasmroundtrip.targets

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
+        <IlasmRoundTripList Include="$(BaseOutputPathWithConfig)\JIT\BBT\Scenario4\Not-Int32\" />
```

Comment:
I am not sure about this global list. Per my earlier comment - I think it would be better to set the ilasmroundtrip property in each test project file, so that it is obvious from the test project file what the test is actually doing.